Essentials/src/messages.properties

```diff
@@ -253,6 +255,7 @@ leatherSyntax=\u00a76Leather color syntax\: color\:<red>,<green>,<blue> eg\: col
 lightningSmited=\u00a76Thou hast been smitten\!
 lightningUse=\u00a76Smiting\u00a7c {0}
 listAfkTag=\u00a77[AFK]\u00a7r
+listAfkHiddenTagFormat=\u00a7f{0}
```

Comment:
In `PlayerList.java`, the `\u00a7f` is appended *after* the display name (`{0}`), but here, it's before the display name. Is this intentional?

Reply:
Oh, I thought changing the player's display name in chat. I'm just looking at the method. :(